blivet/__init__.py

```diff
@@ -127,8 +127,6 @@ def enable_installer_mode():
     from pyanaconda.anaconda_log import program_log_lock
     util.program_log_lock = program_log_lock
 
-    flags.installer_mode = True
-
```

Comment:
Oh, I now see the code is moved to ``osinstall`` in the end which is good enough for now, I think.